### DISCUSSION TITLE
Is downloading detectable?

### DISCUSSION THREAD

**Discussion by GalacticNomad12** [aclap-dev/video-downloadhelper#1315] [Help and Q&A]
### Your question

Hi there,

I'm not very tech-savvy, so any help is appreciated. I would like to save a video from a course on Skillplate. Will the platform or the content creator be notified if I do this?

Thanks in advance!

### Addon & CoApp version and details

version: 9.1.0.44
target: google
channel: stable
lang: en-GB
coapp: {"found":true,"new_version":false,"path":"/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"065eb49b3d374282"}
platform: arm64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "download_directory": "/Users/kristian/dwhelper",
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 3,
    "successfull_dl": 1
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

Downloading by VDH looks the same as playing the video.  I suppose technically, it might be possible for a web site to notice that you are retrieving the video at a speed that is higher than is humanly possible to watch, depending on the speed of the connection being provided by the web server.  But I don't think most web sites bother to collect that level of detailed statistics.  It's more likely they could know that you viewed the video, which would mean you played it on their web site or downloaded it.  But downloading with VDH is not otherwise distinguishable from playing the content on the web page.

It's quite possible the content is protected by DRM.  Check other discussions in this forum by searching with `drm` as your search key.  DRM would prevent any download software from working.

I do hope you test drove VDH before you bought your license.  The license does not unlock otherwise locked features of VDH.  The license does not unlock otherwise locked web sites.  If you tried to download from the site without a license & it didn't work, getting a license wouldn't improve the situation.
